### PR TITLE
adds coverage check to travis ci

### PR DESCRIPTION
**Pull request by rfeistenauer**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,19 +17,76 @@ notifications:
...
+    # Testing Job - Checks against packages in testing stage. Relevant since all these get released together during a sync.
+    - name: "Build + Test with Testing Repo of melodic (http://packages.ros.org/ros-testing/ubuntu)"
+      install:
+        - git clone --depth=1 --branch master https://github.com/ros-industrial/industrial_ci.git .industrial_ci
+      env:
+        - ROS_REPO=testing
+          ROS_DISTRO="melodic"
+      script:
+        - .industrial_ci/travis.sh
+    - name: "Build + Test with Testing Repo of kinetic (http://packages.ros.org/ros-testing/ubuntu)"
```
Comment (martiniil) [inline .travis.yml:59]: Please insert newline and comment as for the other jobs.

File: .travis.yml [hunk 2]
```diff
@@ -15,19 +17,76 @@ notifications:
...
+      script:
+        - .industrial_ci/travis.sh
+        
+    # Testing Job with clang format- Checks against packages in testing stage. Relevant since all these get released together during a sync.
+    - name: "Build + Test with clang format with Testing Repo of melodic (http://packages.ros.org/ros-testing/ubuntu)"
+      install:
+        - git clone --depth=1 --branch master https://github.com/ros-industrial/industrial_ci.git .industrial_ci
+      env:
+        - ROS_REPO=testing
+          ROS_DISTRO="melodic"
```
Comment (martiniil) [inline .travis.yml]: I think for the clang-format check `ROS_REPO` and `ROS_DISTRO` are irrelevant.

**Review by martiniil — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (rfeistenauer) [reply .travis.yml]: at least omitting distro results in an error, we could try removing both, maybe then it works

**Comment by martiniil** [comment]
I like the changes. Not all changes in travis.sh and CMakeLists.txt are related to the coverage check. So please make sure that we get meaningful commit messages on melodic-devel. I think "adds coverage check to travis ci" is a bit too short.

File: .travis.yml (hunk 2 above)
Comment (rfeistenauer) [reply .travis.yml]: We could remove the `ROS_Repo`, Distro is required. Please resolve the conversation if you don't have any other idea how to get rid of the diestro as well

**Review by martiniil — APPROVED** [review]

**Pull request merged**